nullaway/src/main/java/com/uber/nullaway/ErrorBuilder.java

```diff
@@ -0,0 +1,287 @@
+package com.uber.nullaway;
+
+import static com.uber.nullaway.ErrorMessage.MessageTypes.FIELD_NO_INIT;
+import static com.uber.nullaway.ErrorMessage.MessageTypes.METHOD_NO_INIT;
+import static com.uber.nullaway.NullAway.INITIALIZATION_CHECK_NAME;
+import static com.uber.nullaway.NullAway.getTreesInstance;
+
+import com.google.common.base.Joiner;
+import com.google.common.collect.Iterables;
+import com.google.common.collect.Lists;
+import com.google.errorprone.VisitorState;
+import com.google.errorprone.fixes.SuggestedFix;
+import com.google.errorprone.matchers.Description;
+import com.google.errorprone.util.ASTHelpers;
+import com.sun.source.tree.AnnotationTree;
+import com.sun.source.tree.ClassTree;
+import com.sun.source.tree.MethodInvocationTree;
+import com.sun.source.tree.MethodTree;
+import com.sun.source.tree.ModifiersTree;
+import com.sun.source.tree.Tree;
+import com.sun.source.tree.VariableTree;
+import com.sun.source.util.TreePath;
+import com.sun.tools.javac.code.Symbol;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.StreamSupport;
+import javax.annotation.Nullable;
+import javax.lang.model.element.Element;
+
+public class ErrorBuilder {
+
+  private final Config config;
+  private final String checkerName;
```

Comment:
Should this be maybe `suppressionName`?  Looks like it's just used when generating a `@SuppressWarnings`.  Pls also add docs

Reply:
👍Done